rust/opendp_tooling/src/codegen/python.rs

```diff
@@ -252,10 +257,16 @@ fn generate_docstring(func: &Function, hierarchy: &HashMap<String, Vec<String>>)
         }
     );
 
+    let example_path = format!("src/{}/{}.rst", &module_name, &func.name);
```

Comment:
I think this would be better under `examples/`. I think the misc examples files could really clutter up a folder that should primarily have an implementation and the associated proofs.